docs/admin/setup/distributed.rst

```diff
@@ -652,3 +656,117 @@ different ports and a unique instance ID for each instance.
 #. Start the remaining instances.
 
 You should now be able to see all instances in the registry interface at ``http://localhost:8761``.
+
+.. _Running multiple instances locally with Docker:
+
+Running multiple instances locally with Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can also run multiple instances of Artemis locally using Docker. This will start 3 artemis instances, each running
+on a its own container. A load balancer (nginx) will be used to distribute the requests to the different instances. The
+load balancer will be running on a separate container and will be accessible on ports 80/443 of the host system. The
+instances will be registered in the registry service running on a separate container. The instances will use the registry
+service to discover each other and form a hazelcast cluster. Further details can be found in :ref:`Database Cache`. The
+instances will also use a ActiveMQ Artemis broker to synchronize WebSocket messages. Further details can be found in
+:ref:`WebSockets`. In summary, the setup will look like this:
+
+* 3 Artemis instances:
+
+    * artemis-app-node-1: using following spring profile: ``prod,localvc,localci,core,scheduling,docker``
+    * artemis-app-node-2: using following profile: ``prod,localvc,localci,buildagent,core,docker``
+    * artemis-app-node-3: using following profile: ``prod,buildagent``
+* A MySQL database addressable on port 3306 of the host system
+* A Load balancer (nginx) addressable on ports 80/443 of the host system: ``http(s)://localhost``
+* A Registry service addressable on port 8761 of the host system: ``http://localhost:8761``
+* An ActiveMQ broker
+
+
+   .. figure:: distributed/multi-node-setup.drawio.png
+      :align: center
+
+
+
+.. note::
+
+    - You don't have to start the client manually. The client files are served by the Artemis instances and can be
+      accessed through the load balancer on ``http(s)://localhost``.
+
+    - You may run into the following error when starting the containers
+      ``No member group is available to assign partition ownership...``. This issue should resolve itself after a few
+      minutes. Otherwise, you can first start the following containers:
+      ``docker compose -f docker/test-server-multi-node-mysql-localci.yml up mysql jhipster-registry activemq-broker artemis-app-node-1``.
+      After these containers are up and running, you can start the remaining containers:
+      ``docker compose -f docker/test-server-multi-node-mysql-localci.yml up artemis-app-node-2 artemis-app-node-3 nginx``.
+
+
+Linux setup
+"""""""""""
+
+#. (This step is not necessary for MacOS. Here's why: `post <https://stackoverflow.com/a/70385997>`_)
```

Comment:
We should probably tell people that they will be redirected to an external site:
```suggestion
#. (This step is not necessary for MacOS. Here's why: `stackoverflow post <https://stackoverflow.com/a/70385997>`_)
```

Reply:
Removed this line as it was unnecessary (MacOS has its own instructions)